satpy/readers/__init__.py

```diff
@@ -704,7 +704,7 @@ def find_files_and_readers(start_time=None, end_time=None, base_dir=None,
 
 
 def load_readers(filenames=None, reader=None, reader_kwargs=None,
-                 ppp_config_dir=None):
+                 ppp_config_dir=None, file_system=None):
```

Comment:
Should this argument be called `file_system` or `fs`?  It's already called `fs` in [`find_files_and_readers`](https://github.com/pytroll/satpy/blob/40fb438ce9900f481955c117701c457f69ec4231/satpy/readers/__init__.py#L326-L329), and the fsspec classes [`CachingFileSystem`](https://filesystem-spec.readthedocs.io/en/latest/api.html#fsspec.implementations.cached.CachingFileSystem) and [`WholeFileCacheFileSystem`](https://filesystem-spec.readthedocs.io/en/latest/api.html#fsspec.implementations.cached.WholeFileCacheFileSystem) also use `fs` for this parameter, so `fs` would be more consistent with naming elsewhere.

Reply:
I think `file_system` is better, more explicit.

Reply:
Fair point — should we then change it in `find_files_and_readers` as well for consistency?  Otherwise it's a tradeoff between consistency and what is more explicit.

Personally, I prefer `fs`, which is shorter and already very common as an abbreviation for filesystem: nfs, ntfs, zfs, xfs, reiserfs, fsspec, s3fs, [`fspath`](https://docs.python.org/3/library/os.html#os.fspath), and elsewhere.